### PR TITLE
Store arbiter info (Bryan + Malo)

### PR DESCRIPTION
**Pull request by nocommentcode**
This PR Extendeds #158 by enhancing it.

The enter pgn screen now shows a 'current event' section in stead of text boxes
The arbiter can edit the current event, this opens up a sheet where the arbiter can either use a qr code (NOT IMPLEMENTED YET) or manually input in text boxes to store the current events.

When editing the current event, this data is stored in local storage so that it is saved when the app is closed.
The pin is now checked against the arbiter context instead of '1234'

**Setting The event info**

https://user-images.githubusercontent.com/26475724/195227739-d218a980-5fa4-43dc-b0c4-28086551647d.mp4

**Event info display:**

https://user-images.githubusercontent.com/26475724/195227753-f7c26b52-e6f4-48a1-9d31-215caefd6048.mp4

**Pin usage:**

https://user-images.githubusercontent.com/26475724/195227759-36c74ba1-22db-4bcd-85f1-beb4af274ae1.mp4

### CODE REVIEW

**Comment by nocommentcode** [comment]
This is the PR im talking about @gwenllian1

File: packages/TorneloScoresheet/src/style/images.ts
Comment (gwenllian1) [inline packages/TorneloScoresheet/src/style/images.ts:29]: It might be worth having a chat with Harry about adding images as assets because he mentioned that we should be using material icons where possible and they have one for a qr code. But I'm happy to approve it if you think this way is better.

File: packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx
Comment (gwenllian1) [inline packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx:141]: I know it's a placeholder but in our contributing guide it says we should have this stuff in the style file before we merge into main.

File: packages/TorneloScoresheet/src/pages/EnterPgn/EnterPgn.tsx
Comment (gwenllian1) [inline packages/TorneloScoresheet/src/pages/EnterPgn/EnterPgn.tsx:111]: Super minor but it should be iPad (just so that it looks a bit more professional )

File: packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx
Comment (gwenllian1) [inline packages/TorneloScoresheet/src/pages/EnterPgn/EditCurrentEvent.tsx:116]: Could these please be Text Input and QR Input? I think it looks a bit nicer than the lower case i

**Review by gwenllian1 — CHANGES_REQUESTED** [review]
This looks good :) I left a couple of minor comments, the stuff to do with the material icons is entirely optional, I just thought I'd mention it. Another entirely optional thing is I noticed that the link appears a bit above the text: 
image
 
 But I'm on an Android emulator so if it looks good on an iPad then that's all that matters.

File: packages/TorneloScoresheet/src/style/images.ts
Comment (nocommentcode) [reply packages/TorneloScoresheet/src/style/images.ts:29]: yeh i know harry prefers the googel font icons, but david said their policy is normaly to download the assets and use them as svgs, but he did also say they dont really mind either way, here i used the svg files because i need to use the IconTextButton and i cbf changing the component to work with the material icons haha

**Comment by nocommentcode** [comment]
as for the link yes it looks weird for me too, but i cant fix it so i gave up, it was also already like that I just copied it over from where we had it before, so idk if it looks fine on the ipad or not

**Review by gwenllian1 — APPROVED** [review]
Looks good :)

**Pull request merged**
